meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/imx93-11x11-lpddr4x-evk/boot.cmd

```diff
@@ -0,0 +1,29 @@
+echo "Using fdt-freescale_${fdtfile}"
```

Comment:
Here we use `fdt-freescale_` prefix, and in line 11 we assign `freescale_` prefix instead.